R/tune_bayes.R

```diff
@@ -578,13 +578,21 @@ fit_gp <- function(dat, pset, metric, control, eval_time = NULL, ...) {
   }
 
   opts <- list(...)
+
+  withCallingHandlers({
```

Comment:
i chose to do one call to `withCallingHandlers()` around the whole `if` instead of two because it felt better.

no strong opinion either way